File: cours9/global.js
/* Fonction d'appel */
function appelVers(destination) {
  let msg = "Coucou"
  console.log(msg + " Je téléphone depuis " + villeTelFixe + " vers " + destination)
}

/* fonction raccrocher */
function raccrocher() {
  console.log(villeTelFixe + " a raccroché")
}


let villeTelFixe = "Lyon"

// appel vers Lille
appelVers("Lille")

// raccrocher
raccrocher()

File: cours9/global.py
# Fonction d'appel
def appelVers(destination):
    msg = "Coucou"
    print(msg + " Je téléphone depuis " + villeTelFixe + " vers " + destination)


# fonction raccrocher
def raccrocher():
    print(villeTelFixe + " a raccroché")



villeTelFixe = "Lyon"

# appel vers Lille
appelVers("Lille")

# raccrocher
raccrocher()
